index.js

```diff
@@ -11,21 +11,36 @@
 'use strict';
 
 var fs = require('fs');
-var _ = require('lodash');
-var pathExists = require('path-exists');
+var isFunction = require('lodash.isfunction');
```

Comment:
it looked like it had done other things as well but i think it was mostly for edge cases.